### DISCUSSION TITLE
I got this error when I tried to install verstack version 3.0.0 and up

### DISCUSSION THREAD

**Discussion by HISSAOUI** [DanilZherebtsov/verstack#31] [Q&A]
ERROR: Could not find a version that satisfies the requirement tensorflow<=2.11.0,>=2.6.0 (from verstack) (from versions: none)
ERROR: No matching distribution found for tensorflow<=2.11.0,>=2.6.0

can you please help me !

**Comment by DanilZherebtsov** [comment]
> ERROR: Could not find a version that satisfies the requirement tensorflow<=2.11.0,>=2.6.0 (from verstack) (from versions: none) ERROR: No matching distribution found for tensorflow<=2.11.0,>=2.6.0
> 
> can you please help me !

which python version are you using?

**Reply by HISSAOUI** [reply]
I use Python 3.9.16
It seems that the problem is related to the macos M1 where I have to install a compiler before installing lightgbm

**Reply by DanilZherebtsov** [reply]
> I use Python 3.9.16
> 
> It seems that the problem is related to the macos M1 where I have to install a compiler before installing lightgbm

Yeah, there are slight problems with dependencies installation on M macs. Did not test on them. Will sort this out when I get a hold on one of them. 
Meanwhile I suggest to Google the installation of lightgbm and tensorflow on M macs. 

If you discover a quick fix - let me know.
